cleanlab/segmentation/filter.py

```diff
@@ -0,0 +1,180 @@
+# Copyright (C) 2017-2023  Cleanlab Inc.
+# This file is part of cleanlab.
+#
+# cleanlab is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published
+# by the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# cleanlab is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with cleanlab.  If not, see <https://www.gnu.org/licenses/>.
+
+"""
+Methods to find label issues in semantic segmentation datasets, where each pixel in a image receives its own class label.
+
+"""
+
+from cleanlab.experimental.label_issues_batched import find_label_issues_batched
+import numpy as np 
+from typing import Optional, List, Any
+
+def find_label_issues(
+    labels: np.ndarray,
+    pred_probs: np.ndarray,
+    downsample: int = 16,
+    batch_size: int = 10000,
+    n_jobs: Optional[int] = 1,
+    verbose: bool = True,
+    **kwargs) -> np.ndarray:
+    """
+    Identifies potentially bad labels in semantic segmentation datasets using confident learning.
```

Comment:
```suggestion
def find_label_issues(
    labels: np.ndarray,
    pred_probs: np.ndarray,
    *,
    downsample: int = 16,
    batch_size: int = 10000,
    n_jobs: Optional[int] = 1,
    verbose: bool = True,
    **kwargs) -> np.ndarray:
```

Make optiona largs kwargs